frontend/src/views/TripList.vue

```diff
@@ -1,19 +1,21 @@
 <template>
   <div class="full-width">
     <!-- Section for all trips -->
-    <div class="whitespace-top">
-      <h1 class="text-left">Trips</h1>
-      <div v-for="trip in tripsList">
-          <div class="col-xs-3 box">
+    <h1 class="text-left">Trips</h1>
+    <div class="whitespace-top flex-list">
+      <div class="flex-box" v-for="trip in tripsList">
+          <div class="box">
             <div class="item" v-on:click="goToTrip(trip.id)">
-            <img class="photo" v-bind:class="{ 'default-picture': !trip.picture }" :src="trip.picture"/>
+            <!-- Not sure if there's a better way to only render the source tag if the value is not null or blank -->
+            <img v-if="!trip.picture" class="photo" v-bind:class="{ 'default-picture': !trip.picture }"/>
```

Comment:
Not exactly sure about the purpose of this. Was the previously implementation broken? It should have worked even without a `trip.picture`, unless it would error.

Reply:
It's broken now because I added photo URL to the create trip dialog. So now the DB saves an empty string and empty strings in :src render a broken image (as opposed to null, which was apparently fine before?).

Reply:
Ahhh ok. In that case I think you could keep the old implementation and instead of `!trip.picture` you can do `trip.picture == ""`but I'm not sure if that will syntactically work within HTML. I thought that an empty string actually returns false in JavaScript though.

Reply:
The issue isn't the v-bind:class, it's that :src=null renders fine but :src="" renders a broken link. So we can't have the source tag at all, and there's no way to do that conditionally in Vue without duplicating the entire tag.

Reply:
Oh right, ops I forgot that. Nevermind to what I said before.